modules/gapi/perf/common/gapi_render_perf_tests.hpp

```diff
@@ -0,0 +1,56 @@
+// This file is part of OpenCV project.
+// It is subject to the license terms in the LICENSE file found in the top-level directory
+// of this distribution and at http://opencv.org/license.html.
+//
+// Copyright (C) 2020 Intel Corporation
+
+
+#ifndef OPENCV_GAPI_RENDER_PERF_TESTS_HPP
```

Comment:
Why do we need this file ?

Can we move all stuff from here to corresponding *.cpp file ?

Reply:
Removed